lib/vc.js

```diff
@@ -162,7 +310,85 @@ function _getId(obj) {
   return obj.id;
 }
 
-async function _checkCredential({credential}) {
+/**
+ * @param presentation
+ * @throws {Error}
+ * @private
+ */
+function _checkPresentation(presentation) {
+  if(!presentation['@context']) {
+    throw new Error('"@context" property is required.')
+  }
+
+  // check context cardinality
+  if(jsonld.getValues(presentation, '@context').length <= 1) {
```

Comment:
I'd pick one style and make cred/pres `@context` and `type` checks use either `<= 1` or `< 2`.

Reply:
Thanks, fixed.